common/__init__.py

```diff
@@ -41,6 +44,26 @@ def prepare_server_properties(properties: List[str]) -> List[str]:
     none_empty = list(filter(None, stripped))
     return none_empty
 
+def deep_get(dictionary: Dict[str, Any], *keys):
+    return reduce(lambda d, key: d.get(key) if d else None, keys, dictionary)
+
+PHANTOM_HTML = """
+<style>div.phantom {{font-style: italic; color: #00ff73}}</style>
+<div class='phantom'>{}</div>"""
+
+def decoraction_to_phantom(option: Dict[str, Any], view: sublime.View) -> Optional[sublime.Phantom]:
+    end = Point(deep_get(option, 'range', 'end', 'line'), deep_get(option, 'range', 'end', 'character'))
+    region = range_to_region(Range(end, end), view)
```

Comment:
You don't really need to fiddle with points:
```suggestion
    region = range_to_region(Range.from_lsp(option['range'], view)
```

Although you would then have to change it further into

```
region.a = region.b  # make the start point equal to the end point
```

It's up to you what you find more elegant :)